feature/dashboard/src/main/kotlin/com/chipichipi/dobedobe/feature/dashboard/component/DashboardEditMode.kt

```diff
@@ -49,25 +52,41 @@ internal fun DashboardEditMode(
     modifier: Modifier = Modifier,
 ) {
     var selectedPhotoId by remember { mutableStateOf<Int?>(null) }
+    val context = LocalContext.current
+    var showCropView by remember { mutableStateOf(false) }
+    val coroutineScope = rememberCoroutineScope()
+    val cropView = remember {
+        CropImageView(context).apply {
+            isAutoZoomEnabled = false
+            cropShape = CropImageView.CropShape.RECTANGLE
+            setFixedAspectRatio(true)
+        }
+    }
```

Comment:
```suggestion
    val cropView = remember(context) {
        CropImageView(context).apply {
            isAutoZoomEnabled = false
            cropShape = CropImageView.CropShape.RECTANGLE
            setFixedAspectRatio(true)
        }
    }
```

제가 컴포즈에서는 ViewSystem 을 사용 안해봐서 잘은 모르지만, context 가 바뀌어도 문제가 생기지 않나요?!

Reply:
[3799d6f](https://github.com/Nexters/dobedobe-android/pull/57/commits/3799d6f05ed2fe3b9315265deb84817e77a1fdb2)
넣어주는 게 좋을 것 같네요 반영했습니다!